posts/new-ep-sat.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: New Ep Drops Sat
+description: "Check out Gyst Rewind this Saturday 8.19.23 for an excellent conversation with B.I.G.G (@biggstreetz on IG) "
+date: Aug 17 2023
+---
+
+New episode with [@biggstreetz](https://www.instagram.com/biggstreetz/) dropping Saturday! [#podcast](https://www.instagram.com/explore/tags/podcast/?hl=en) [#Gystrewind](https://www.instagram.com/gyst.rewind/) [#Minneapolis](https://www.instagram.com/explore/locations/214063175/minneapolis-minnesota/?hl=en)
```

Comment:
```suggestion
New episode with <a href="https://www.instagram.com/biggstreetz/" target="_blank">@biggstreetz</a> dropping Saturday! [#podcast](https://www.instagram.com/explore/tags/podcast/?hl=en) [#Gystrewind](https://www.instagram.com/gyst.rewind/) [#Minneapolis](https://www.instagram.com/explore/locations/214063175/minneapolis-minnesota/?hl=en)
```

see if this works?

got it from [here](https://www.eddymens.com/blog/how-to-make-a-markdown-link-open-in-another-tab#:~:text=You%20just%20add%20the%20target,in%20a%20different%20browser%20tab)

Reply:
we can try this later; i know that MDX works with the current format. it may need to be updated tho for best practices